tests/gamma/csm_observability_test.py

```diff
@@ -45,39 +92,383 @@ def setUpClass(cls):
         super().setUpClass()
         cls.metric_client = cls.gcp_api_manager.monitoring_metric_service("v3")
 
+    # A helper function to make the cloud monitoring API call to query metrics
+    # created by this test run.
+    def _query_metrics(self, metric_names, filter_str, interval):
+        results = {}
+        for metric in metric_names:
+            response = self.metric_client.list_time_series(
+                name=f"projects/{self.project}",
+                filter=filter_str
+                % {
+                    "metric": metric,
+                    "grpc_method": _GRPC_METHOD_NAME,
+                },
+                interval=interval,
+                view=monitoring_v3.ListTimeSeriesRequest.TimeSeriesView.FULL,
+            )
+            time_series = []
+            logger.debug("Metric: %s", metric)
+            for series in response:
+                logger.debug(" Metric Labels:")
+                for label_key, label_value in series.metric.labels.items():
+                    logger.debug(
```

Comment:
Instead of multiple `logger.debug()`, just make list, push each line into it, and do one 
```py
logger.debug('\n'.join(my_messages))
```

At this point we, can even do `logger.info`

Reply:
Done